### PR TITLE
Add an article about what Kinto can do for me

### PR DESCRIPTION
**Pull request by almet**
I was thinking about the fact that we probably miss a good introduction article about what Kinto is, so I made this thing. Reviews and comments are very welcome, this can benefit from some enhancement before it is published!

### CODE REVIEW

File: content/2015.12.what-can-kinto-do.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
+What can Kinto do for me?
+#########################
+
+:date: 2015-12-26
+:slug: what-can-kinto-do-for-you
+:lang: en
+:url: what-can-kinto-do-for-you
+:summary: Kinto, the project we're working on can do a bunch of different
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: the project we are working on**,**

File: content/2015.12.what-can-kinto-do.rst [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+          things, bit it's sometimes hard to see what it's really good at, and
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: but it is

File: content/2015.12.what-can-kinto-do.rst [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+Kinto with React, Angular, JQuery (in the JavaScript world) or any framework in
+any language. Kinto abstracts the synchronisation of the application's data.
+
+Some examples
+#############
+
+Let's take some examples. A few weeks ago we prepared a bunch of demos
+showcasing what Kinto can do and how easy it is. In order to do that, we
+decided to take a few applications we liked that are usually not doing
+synchrnisation sharing. There are many out there, and we chose just a few of
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: synchronisation and sharing

File: content/2015.12.what-can-kinto-do.rst [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+them:
+
+- **A JQuery calendar application** which let you add new events and see them
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst:43]: We should add links there.

File: content/2015.12.what-can-kinto-do.rst [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  on a calendar.
+- **A leaflet map** where you can add new markers and move them easily.
+- **A TODO-list application**, because we actually really like TODO lists.
+
+And then we decided to make them collaborative apps, using Kinto. The goal here
+was to spend the least time possible plugging Kinto, in order to see what was
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: plugging in Kinto

File: content/2015.12.what-can-kinto-do.rst [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+of Kinto (for instance some take advantage of the push notifications while some
+others don't), but overall that was pretty exciting.
+
+Here are some links if you want to try out the demos yourself:
+
+* `Kinto.js + Leaflet <http://leplatrem.github.io/kinto-demo-leaflet/>`_ which
+  requires service workers activated (only works in Firefox Nightly these
+  days).
+* `Kinto.js + fullCalendar <http://leplatrem.github.io/kinto-demo-calendar/>`_
+* `Kinto.js TodoMVC <http://natim.github.io/kinto-demo-todomvc>`_
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: We should add the cliquet-pusher demo as well as the Kinto-Telegram-Wall.

[0] http://leplatrem.github.io/cliquet-pusher/
[1] http://leplatrem.github.io/kinto-telegram-wall/

File: content/2015.12.what-can-kinto-do.rst [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+Hmm, and what about my use-case?
+################################
+
+The idea here is mainly to show you what's possible to do with very little
+efforts using the tooling we have put together.  These examples are all relying
+on the `Kinto.js <https://kintojs.readthedocs.org>`_ but ultimately you don't
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: Kinto.js client

File: content/2015.12.what-can-kinto-do.rst [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+need it. it's just here to make it really easy for you to integrate with the
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: It is

File: content/2015.12.what-can-kinto-do.rst [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+HTTP APIs exposed by Kinto.
+
+So, what Kinto can do for you?
+
+Synchronize
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: Synchronise or Synchronize (before you used synchronisation)

File: content/2015.12.what-can-kinto-do.rst [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+===========
+
+If there was one only thing to chose, that would be this one. Kinto can
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: to choose

File: content/2015.12.what-can-kinto-do.rst [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+
+The cool thing with this is that you can access this data using an HTTP (REST)
+API, that speaks JSON. We tried very hard to make the protocol as easy as
+possible, and to be honest we're pretty happy with our current design.
+
+Permissions
+===========
+
+Sometimes, you want your data to remain yours, and some other times you want to
+share it with friends. It really depends what's the data. One of the reasons we
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: what is the data.

File: content/2015.12.what-can-kinto-do.rst [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+built Kinto was to control who has access to what.
+
+Notifications
```
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst]: Real-Time Notifications ?

**Comment by Natim** [comment]
This is really great !!

File: content/2015.12.what-can-kinto-do.rst [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+decided to take a few applications we liked that are usually not doing
+synchronisation and sharing. There are many out there, and we chose just a few of
+them:
+
+- **A JQuery calendar application** which let you add new events and see them
+  on a calendar.
+- **A leaflet map** where you can add new markers and move them easily.
+- **A TODO-list application**, because we actually really like TODO lists.
+
+And then we decided to make them collaborative apps, using Kinto. The goal here
```
Comment (michielbdejong) [inline content/2015.12.what-can-kinto-do.rst:48]: 'collaborative' sort of implies that one data set is shared by multiple people - is that what you mean?
You could also refer to 'task continuation': start on one device, and continue editing the same data on another.

File: content/2015.12.what-can-kinto-do.rst [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+was to spend the least time possible plugging in Kinto, in order to see what was
+the real potential of it.
+
+We ended-up with a few cool demos. Not all of them are using the same features
```
Comment (michielbdejong) [inline content/2015.12.what-can-kinto-do.rst]: Nit: s/ended-up/ended up

File: content/2015.12.what-can-kinto-do.rst [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+
+* `Kinto.js + Leaflet <http://leplatrem.github.io/kinto-demo-leaflet/>`_
+  with offline-first maps which requires service workers activated
+  (only works in Firefox Nightly these days).
+* `Kinto.js + Leaflet + Pusher <http://leplatrem.github.io/cliquet-pusher/>`_
+  with realtime multi device map markers synchronisation using Pusher.
+* `Kinto.js + fullCalendar <http://leplatrem.github.io/kinto-demo-calendar/>`_
+  with realtime multi device synchronisation using Pusher and Firefox Account integration.
+* `Kinto.js + Telegram Bot + Pusher <http://leplatrem.github.io/kinto-telegram-wall/>`_
+  a realtime wall. That receive message from a Telegram bot and automatically update the
```
Comment (michielbdejong) [inline content/2015.12.what-can-kinto-do.rst]: Nit: 'a realtime wall that receives messages from a Telegram bot and automatically updates ...'

File: content/2015.12.what-can-kinto-do.rst [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+Hmm, and what about my use-case?
+################################
+
+The idea here is mainly to show you what is already possible with very little
+efforts using the tooling we have put together. These examples are all relying
+on the `Kinto.js <https://kintojs.readthedocs.org>`_ client but ultimately you don't
+need it. It's just here to make it really easy for you to integrate with the
+HTTP APIs exposed by Kinto.
+
+So, what Kinto can do for you?
```
Comment (michielbdejong) [inline content/2015.12.what-can-kinto-do.rst]: Nit: s/Kinto can/can Kinto

File: content/2015.12.what-can-kinto-do.rst (hunk 13 above)
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst:48]: In that case it is collaborative because the demo apps share data between all the users.

**Comment by Natim** [comment]
Thank you @michielbdejong

**Pull request merged**